src/components/Input/interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface SearchInputProps {
+  suggestions: Array<string>;
```

Comment:
Those are more like `items` in a sense: Problems and Tags, only after interacting with the input search you filter them and then the limited group of them become `suggestions` to your search term.